CHANGELOG.md

```diff
@@ -1,5 +1,80 @@
 # Chef Server Changelog
 
+## [12.15.0](https://github.com/chef/chef-server/tree/12.15.0) (2017-04-27)
+[Full Changelog](https://github.com/chef/chef-server/compare/12.14.0...12.15.0)
+
+## 12.15.0 (2017-04-27)
+### Components
+New Components
+* server-complete ()
+
+Updated Components
+* cacerts (2016-04- -> 2017-01-)
+* openssl (1.0.2j -> 1.0.2k)
+* omnibus-ctl (43b10417 -> 76347dd8)
+* knife-opc (00a6866c -> 30b6cd85)
+* knife-ec-backup (36fbecfd -> 69b50524)
+* liblzma (5.2.2 -> 5.2.3)
+
+
+### Contributors
+* Steven Danna
+* Salim Afiune
+* Mark Anderson
+* Marc Paradise
+* Jaymala Sinha
+* Ryan Cragun
+* Stephen Delano
+* Stephan Renatus
+* Thom May
+
+**Implemented enhancements:**
+
+- LDAP: Synthesize a displayname from surname, givenname [\#151](https://github.com/chef/chef-server/issues/151)
+- Mapping Capability Between LDAP and Chef Server 12 Attributes [\#104](https://github.com/chef/chef-server/issues/104)
+- Enhance API to handle global groups in local contexts [\#1159](https://github.com/chef/chef-server/pull/1159) ([markan](https://github.com/markan))
+
+**Closed issues:**
+
+- t [\#1234](https://github.com/chef/chef-server/issues/1234)
+- org-create not executed successfully [\#1231](https://github.com/chef/chef-server/issues/1231)
+- Failure upgrading RabbitMQ from Migration 1.30 to 1.31 [\#1220](https://github.com/chef/chef-server/issues/1220)
+- chef-sever 12.14 HA erchef keeps looking for rabbitmq process... [\#1212](https://github.com/chef/chef-server/issues/1212)
+- Installing opscode-reporting on chef-server v12.17.44 breaks chef irreparably [\#1208](https://github.com/chef/chef-server/issues/1208)
+- Allow setting LDAP "displayName" attribute in Chef Server [\#800](https://github.com/chef/chef-server/issues/800)
+- LDAP user name mojibake [\#675](https://github.com/chef/chef-server/issues/675)
```

Comment:
Maybe we should get rid of the closed issues that didn't show up as merged pull requests, like `t`